### ISSUE TITLE
Only place holder is used as visual label for "Key" and "Value" edit field: A11y_Azure Tools Storage Explorer_Blob container_Upload file_Label and instructions

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8377]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #WCAG3.3.2; #FTP; #Label and Instructions; #Closed; #Reressed:01-27-25;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch the Azure Tools Storage Explorer.
2. TAB till "Blob container" in tree view and press ENTER key to expand it.
3. TAB till any blob container and press ENTER key.
4. TAB till "Upload" button and press ENTER key.
5. Press Down arrow key to upload file menu item and press ENTER key
6. TAB till "Key" and "Value" edit field.
7. Observe that only place holder is used as visual label for "Key" and "Value" edit field

## Actual Experience

Only place holder is used as visual label for "Key" and "Value" edit field

## Expected Experience

Place holder should not be the only way for visual label as soon as we start interacting with edit label should be visible that time also. 
## User Impact:

User will face difficulty if they start interacting with edit field and label will not visible. 

## Attachment
![Image](https://github.com/user-attachments/assets/4e2cde93-6d28-4354-9444-efc504366811)

**Comment by craxal** [comment]
In other words, we should have an actual label for Key fields and Value fields.

**Comment by craxal** [comment]
@kapilvaishna Fix has been merged. Please verify with new build.

**Comment by kapilvaishna** [comment]
Issue is repro in 1.37.0 version, please find attachment for reference.

![Image](https://github.com/user-attachments/assets/958458d7-2e42-4384-8b97-3c5df84d8ba5)


![Image](https://github.com/user-attachments/assets/2142634a-4d90-4a64-9ded-543907dda6bf)

**Comment by craxal** [comment]
@kapilvaishna Which branch did you get that build from? You should be getting builds from the main branch.

**Comment by kapilvaishna** [comment]
@craxal I am downloading the latest version from here: [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).. I am unsure about the main branch.

Could you please confirm if this is correct?

**Comment by craxal** [comment]
@kapilvaishna Let's take this offline.

**Comment by kapilvaishna** [comment]
issue is not repro in below build so closing the bug.

Application: Microsoft azure storage explorer
Version: 1.37.0 (104)

![Image](https://github.com/user-attachments/assets/c899b596-a9d3-43f0-95ff-2f8f6afdad44)
